mandatory/1-fix-functions.js

```diff
@@ -33,20 +32,20 @@ function greaterThan10(num) {
 function sortArray(letters) {
   let sortedLetters = letters;
 
-  return sortedLetters;
+  return sortedLetters.sort();
```

Comment:
The change is right, but I would do it on line 33 instead.
 So on line 33, you are taking the `letters` variable, you will `sort` it and then store the result in `sortedLetters`

Reply:
@louisechow it is as it follows : 

```
function sortArray(letters) {
  let sortedLetters = letters.sort();

  return sortedLetters;
}
```